source/includes/administration/_environments.md

```diff
@@ -482,6 +482,16 @@ curl -X DELETE "https://cloudmc_endpoint/v2/environments/[environment-id]/member
 
 ```
 
+```shell
+# Response body example
+```
+```json
+{
+  "taskId": "a82113b9-7f76-4354-ae27-b18b0204d50f",
+  "status": "PENDING"
+}
+```
```

Comment:
This looks good. We could also mention above that it is an asynchronous API, so we could link the Tasks section. Should we update the resource endpoint above to `/v2/environments` ?

Reply:
the end point is already v2, I added a detail description